app/api/routes.py

```diff
@@ -99,14 +98,7 @@ def apikey():
 
 # Helpers
 def get_resource(id):
-    resource = None
-    try:
-        resource = Resource.query.get(id)
-
-    except NoResultFound as e:
-        print_tb(e.__traceback__)
-        logger.exception(e)
-        return redirect('/404')
+    resource = Resource.query.get(id)
```

Comment:
In all the places that I removed the NoResultsFound Exception, it's because get() cannot throw noresultsfound. The .one() function can though.
[Heres the link to the docs](https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.get)
Let me know if I'm crazy for thinking that.

I want to add a catch for ObjectDeletedError but I'm not sure how to even throw the error yet. 
If someone knows how to properly do that, let me know and I can make that change. Tht would improve the coverage quite a bit too

Reply:
Can you explain your issue with ObjectDeletedError?

We currently don't have route to delete, and per @aaron-suarez we wouldn't physically delete it but tag the row as non-available and then update the query/ serializer to respect that.

Reply:
Well, then we can probably just remove the try blocks all together on those functions. Then we wont need to test those non existent exceptions.

My only "problem" with it was that it is the only exception that the get() method can throw, so it would be the only exception we would need to catch.

Reply:
Oh that makes sense. But yes, at this time we do not have a way to delete anything. If we do add a route to delete resources/languages/categories, then we'll need to make sure we're catching that. But even so, I think we would just mark as deleted in the db and it would just filter "deleted" results out rather than throwing that error. Either way, we'll address it if and when it comes up. For now, I don't think we need to worry about it